pom.xml

```diff
@@ -220,7 +220,41 @@
                     </plugin>
                 </plugins>
             </build>
-      </profile>
+        </profile>
+        <profile>
+            <id>checkstyle</id>
+            <activation>
+                <property>
+                    <name>!no-checkstyle</name>

```

Comment:
@asoldano No functional objections to this, but ... if the only reason for this is an option to disable checkstyle (`-Dno-checkstyle`), the same behaviour can be reached with `-Dcheckstyle.skip=true` (see [checkstyle docs](https://maven.apache.org/plugins/maven-checkstyle-plugin/checkstyle-mojo.html#skip)) == no regression, no improvement, more lines of pom file ...

Reply:
eh, I did not notice that, thanks. I'll remove the commit with this profile